frontend/src/entities/project/detail/project-detail.tsx

```diff
@@ -62,10 +62,10 @@ function ProjectDetail () {
             .catch(() => {
                 navigate('/404');
             });
-        dispatch(setItemsForProjectName(projectName));
+        dispatch(setItemsForProjectName({ 'project': projectName, 'enabledServices': applicationConfig.configuration.EnabledServices }));
```

Comment:
Why are we dispatching this here? Shouldn't this be dispatched when the app config is updated? The reason we have these dispatches here is to add the project name when we end up on a project page.

Additionally, since the reducer should have access to the state, it should be able to access the current appConfig which is part of the store and shouldn't need this passed in unless I am missing someting.